DataConnectors/BETTER MTD.json

```diff
@@ -0,0 +1,126 @@
+{
+    "id": "BetterMTD",
+    "title": "BETTER Mobile Threat Defense (MTD)",
+    "publisher": "BETTER Mobile",
+    "descriptionMarkdown": "The BETTER MTD Connector allows Enterprises to connect their Better MTD instances with Azure Sentinel, to view their data in Dashboards, create custom alerts, use it to trigger playbooks and expands threat hunting capabilities. This gives users more insight into their organization's mobile devices and ability to quickly analyze current mobile security posture which improves their overall SecOps capabilities.",
+    "graphQueries": [
+        {
+            "metricName": "Total incidents received",
+            "legend": "BetterMTDIncidentLog_CL",
+            "baseQuery": "BetterMTDIncidentLog_CL"
+        },
+        {
+            "metricName": "Total devcie enrolled",
+            "legend": "BetterMTDDeviceLog_CL",
+            "baseQuery": "BetterMTDDeviceLog_CL"
+        },
+        {
+            "metricName": "Total netflpw traffic recieved",
```

Comment:
typo? netflow

Reply:
Yes, thanks for pointing that out, I have fixed that.